terraform/ecs_script_task/templates/default.json.template

```diff
@@ -4,7 +4,7 @@
     "essential": true,
     "image": "${app_uri}",
     "memory": ${memory},
-    "name": "app",
+    "name": "${name}",
```

Comment:
Why has this become a variable?

Reply:
We need to pass this along to the run task lambda and i prefer this approach to hardcoding the value.